src/diffusers/loaders.py

```diff
@@ -1236,9 +1310,12 @@ def _convert_kohya_lora_to_diffusers(cls, state_dict):
         unet_state_dict = {}
         te_state_dict = {}
         network_alpha = None
+        unloaded_keys = []
```

Comment:
The changes in this method look good to me code-wise, but I'm not sure the best way to verify these changes are correct. Could we follow up with tests for converting the kohya checkpoints?

Reply:
https://github.com/huggingface/diffusers/issues/4247

Reply:
Ah, now I see `test_load_unload_load_lora`. Does this test the kohya checkpoint format? If so could we update the name of it to note that that's the type of checkpoint it's testing?

Reply:
It was mainly for testing (@sayakpaul can correct me on this as the original author) the load+unload+load behavior since patching in the initial versions had a side effect which went uncaught without that test (but it also uses a civit ai checkpoint so i think it partially tests this as well).

Reply:
Yeah. Let me elaborate that in the test method. 

@williamberman there's another `test_unload_lora()` fast test too that is in the diffusers format. So, should be okay, I guess? 

https://github.com/huggingface/diffusers/blob/428dbfecd935875dee8fedef393827f54e026e27/tests/models/test_lora_layers.py#L409

Reply:
Fixed in https://github.com/huggingface/diffusers/pull/4147/commits/d05a8d2584d2bba22254ef77eac3580eb29a4f4f